src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -616,6 +616,11 @@ namespace Microsoft.FSharp.Core
         /// <summary>Indicates one or more adjustments to the compiled representation of an F# type or member</summary>
         member Flags : CompilationRepresentationFlags
 
+    module internal ExperimentalAttributeMessages = begin
```

Comment:
@KevinRansom This was required to allow the use of the `RequiresPreview` flag outside of prim-types. What do you think?

Reply:
Yes this change is good, the message needs to be visible accross FSharp.Core.